tests/api/mpi/testexec.in

```diff
@@ -2,6 +2,7 @@
 
 export PYTHONPATH=$PYTHONPATH:@prefix@/include/python
 export SST_LIB_PATH=@prefix@/lib:$SST_LIB_PATH
+export LD_LIBRARY_PATH=@prefix@/lib:$LD_LIBRARY_PATH
```

Comment:
Should be like sstmac-check.in:

export PYTHONPATH=@abs_top_srcdir@/python:$PYTHONPATH
export SST_LIB_PATH=@abs_top_builddir@/sstmac/install/.libs
export DYLD_LIBRARY_PATH=@abs_top_builddir@/sstmac/install/.libs:$DYLD_LIBRARY_PATH
export DYLD_LIBRARY_PATH=@abs_top_builddir@/sst-dumpi/dumpi/libundumpi/.libs:$DYLD_LIBRARY_PATH
export LD_LIBRARY_PATH=@abs_top_builddir@/sstmac/install/.libs:$LD_LIBRARY_PATH
export LD_LIBRARY_PATH=@abs_top_builddir@/sst-dumpi/dumpi/libundumpi/.libs:$LD_LIBRARY_PATH

Reply:
Okay, except with `prefix` instead of build?

Reply:
Yes, for testexec it should be wherever things get installed.
Don't forget the sst-dumpi path, libmacro depends on libundmpi

Reply:
Ok, dumpi and macro prefix are the same. Resolved.